addons/sale_timesheet/wizard/sale_make_invoice_advance.py

```diff
@@ -46,6 +46,6 @@ def _create_invoices(self, sale_orders):
             return sale_orders.with_context(
                 timesheet_start_date=self.date_start_invoice_timesheet,
                 timesheet_end_date=self.date_end_invoice_timesheet
-            )._create_invoices(final=self.deduct_down_payments)
+            )._create_invoices(final=True)
```

Comment:
Is this intended ?

Reply:
The field was always True (appears to be leftover from some time ago when it was used) ... so I removed it